### PR TITLE
Add logging clone for issue #4

### PR DESCRIPTION
**Pull request by shanep**
### What are you trying to accomplish?

Added a logging statement to indicate the number of repos cloned and if there are more pages left to clone

### What are you trying to accomplish?

Finishes implementing #4

### What approach did you choose and why?

Leverage the code that was written to spawn multiple channels to give more detailed logging

### What should reviewers focus on?

If this completes #4 :)

### CODE REVIEW

**Review by zrdaley — DISMISSED** [review]
Thank you for adding tests!

File: cmd/gh-classroom/clone/student-repos/student-repos.go [hunk 1]
```diff
@@ -101,6 +101,13 @@ func NewCmdStudentRepo(f *cmdutil.Factory) *cobra.Command {
 					fmt.Printf("Skip existing repo: %v use gh classroom pull to get new commits\n", clonePath)
```
Comment (zrdaley) [inline cmd/gh-classroom/clone/student-repos/student-repos.go:101]: nit: even if we skip all repos, we will still get something like:
```
Skip existing repo: ...
Skip existing repo: ...
Cloned 2 repos.
```

Comment (shanep) [reply cmd/gh-classroom/clone/student-repos/student-repos.go:103]: Nice catch :) Updated the logging to not count repos that were skipped!

**Review by zrdaley — APPROVED** [review]
Thank you for updating to count!

**Comment by shanep** [comment]
Hi @zrdaley was there anything else needed on this pull request to get it fully merged?

**Comment by zrdaley** [comment]
https://github.com/github/gh-classroom/releases/tag/v0.1.12 should contain this change!

**Pull request merged**